opencensus/spanner/main.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Sample spanner contains a sample application that
+// uses Google Spanner Go client, and reports metrics
+// and traces for the outgoing requests.
+package main
+
+import (
+	"context"
+	"log"
+
+	"cloud.google.com/go/spanner"
+	ss "go.opencensus.io/exporter/stats/stackdriver"
+	ts "go.opencensus.io/exporter/trace/stackdriver"
```

Comment:
These package names aren't ideal. Is there some better one?

Reply:
I wouldn't focus too much on this, we are merging these packages into one package soon. https://github.com/census-instrumentation/opencensus-go/issues/312.